### ISSUE TITLE
Desktop build broken

### ISSUE THREAD

**Issue by ChristopherHackett** [OWASP/threat-dragon#624]
**Describe the bug**
Between 54c1feb...2414d09 `Build application` step of `desktop e2e tests` job has started to fail. This suggesting an issue with the electron build process has been added between these points. 

```
$ npm run build:desktop -- --publish=never
[...]

 DONE  Build complete. The dist-desktop\bundled directory is ready to be deployed.
 INFO  Check out deployment instructions at https://cli.vuejs.org/guide/deployment.html
       
-  Bundling main process...
 ERROR  Failed to compile with 1 errors4:49:18 PM

 error  

background.js from Terser
Unexpected character '#' [background.js:45911,4]
    at ne (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:19592)
    at m (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:20906)
    at Object.P [as input] (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:26635)
    at a (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:28057)
    at s (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:28111)
    at q (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:44177)
    at D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:32229
    at D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:29030
    at N (D:\a\threat-dragon\threat-dragon\td.vue\node_modules\.pnpm\terser@4.8.1\node_modules\terser\dist\bundle.min.js:1:38557)

```

**Expected behaviour**
`main` branch jobs should pass. 

**Environment**
Github Actions

**To Reproduce**
```
git checkout main
pnpm install && pnpm build:desktop  
```
**Any additional context, screenshots, etc**
![image](https://user-images.githubusercontent.com/1075126/225646332-10e55712-6d96-491a-b471-3a5357c0c9df.png)

**Comment by ChristopherHackett** [comment]
Checking locally (`pnpm install && pnpm build:desktop `) shows 
🚫 2414d09 rollback version of got
🚫 1fbe97e fixup electron post install
🚫 7a9a09d apply security updates
✅ 54c1feb run browserstack only from parent repo

So the break appeaser to have in fact occurred between 54c1feb...7a9a09d

**Comment by jgadsden** [comment]
can I assign this to you @ChristopherHackett ? I agree with your analysis

**Comment by jgadsden** [comment]
Hello @ChristopherHackett - this is becoming critical, I can look at this if you like?

**Comment by jgadsden** [comment]
Weirdly it was version 4 of rimraf that was causing the trouble - not directly but something to do with dependencies I guess. Anyhow, all fixed in commit 5a5f89d7d0eb803b4ae070af068117f28b8a989d
